internal/services/ingest_test.go

```diff
@@ -215,3 +216,166 @@ func TestIngestPayments(t *testing.T) {
+	mockAppTracker := apptracker.MockAppTracker{}
+	mockRPCService := RPCServiceMock{}
+	mockRouter := tssrouter.MockRouter{}
+	tssStore, _ := tssstore.NewStore(dbConnectionPool)
+	ingestService, _ := NewIngestService(models, "ingestionLedger", &mockAppTracker, &mockRPCService, &mockRouter, tssStore)
+	srcAccount := keypair.MustRandom().Address()
+	destAccount := keypair.MustRandom().Address()
+	usdIssuer := keypair.MustRandom().Address()
+	eurIssuer := keypair.MustRandom().Address()
+
+	t.Run("test_op_path_payment_send", func(t *testing.T) {
+		err := models.Account.Insert(context.Background(), srcAccount)
+		require.NoError(t, err)
+
+		path := []txnbuild.Asset{
+			txnbuild.CreditAsset{Code: "USD", Issuer: usdIssuer},
+			txnbuild.CreditAsset{Code: "EUR", Issuer: eurIssuer},
+		}
+
+		pathPaymentOp := txnbuild.PathPaymentStrictSend{
+			SourceAccount: srcAccount,
+			Destination:   destAccount,
+			DestMin:       "9",
+			SendAmount:    "10",
+			SendAsset:     txnbuild.NativeAsset{},
+			DestAsset:     txnbuild.NativeAsset{},
+			Path:          path,
+		}
+		transaction, err := txnbuild.NewTransaction(txnbuild.TransactionParams{
+			SourceAccount: &txnbuild.SimpleAccount{
+				AccountID: keypair.MustRandom().Address(),
+			},
+			Operations:    []txnbuild.Operation{&pathPaymentOp},
+			Preconditions: txnbuild.Preconditions{TimeBounds: txnbuild.NewTimeout(10)},
+		})
+		require.NoError(t, err)
+
+		signer := keypair.MustRandom()
+		errSigner := models.Account.Insert(context.Background(), signer.Address())
+		require.NoError(t, errSigner)
+
+		signedTx, err := transaction.Sign(network.TestNetworkPassphrase, signer)
+		require.NoError(t, err)
+
+		txEnvXDR, err := signedTx.Base64()
+		require.NoError(t, err)
+
+		ledgerTransaction := entities.Transaction{
+			Status:           entities.SuccessStatus,
+			Hash:             "efgh",
+			ApplicationOrder: 1,
+			FeeBump:          false,
+			EnvelopeXDR:      txEnvXDR,
+			ResultXDR:        "AAAAAAAAAGQAAAAAAAAAAQAAAAAAAAANAAAAAAAAAAAAAAAAXF0V022EgeGIo6QNVbMVvHdxvvl2MfZcVZUJpfph+0QAAAAAAAAAAAX14QAAAAAA",
+			Ledger:           1,
+		}
+
+		ledgerTransactions := []entities.Transaction{ledgerTransaction}
+
+		err = ingestService.ingestPayments(context.Background(), ledgerTransactions)
+
+		payments, _, _, err := models.Payments.GetPaymentsPaginated(context.Background(), srcAccount, "", "", data.ASC, 1)
+		require.NoError(t, err)
+		require.NotEmpty(t, payments, "Expected at least one payment")
+		assert.Equal(t, payments[0].TransactionHash, *&ledgerTransaction.Hash)
+		assert.Equal(t, payments[0].SrcAmount, int64(100000000))
+		assert.Equal(t, payments[0].SrcAssetType, xdr.AssetTypeAssetTypeNative.String())
+		assert.Equal(t, payments[0].ToAddress, destAccount)
+		assert.Equal(t, payments[0].FromAddress, srcAccount)
+		assert.Equal(t, payments[0].SrcAssetCode, "XLM")
+		assert.Equal(t, payments[0].DestAssetCode, "XLM")
+	})
+}
+
+func TestIngestPathPaymentsReceive(t *testing.T) {
```

Comment:
Same as above. Can you make this a unit test under the TestIngestPayments function?

Reply:
makes sense, I removed this and all instances you mentioned in 0b1637c33563664ae43cf0deb152bbdbbc9ba0ea